nixos/modules/services/misc/jellyseerr.nix

```
@@ -0,0 +1,68 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.jellyseerr;
+in
+{
+  options = {
```

You can add yourself as maintainer for this module, this way you'll get pinged for future PR regarding this module.

```suggestion
  meta.maintainers = [ lib.maintainers.camillemndn ];

  options = {
```